websockets.md

```diff
@@ -345,12 +348,16 @@ websocket.send(JSON.stringify(message));
 ## Keep a connection alive
 {: #WSkeep}
 
-The service terminates the session and closes the connection if an inactivity or session timeout is reached:
+The service terminates the session and closes the connection if an inactivity or session timeout occurs:
+
+-   An *inactivity timeout* occurs if audio is being sent by the client but the service detects no speech. The inactivity timeout is 30 seconds by default. You can use the `inactivity_timeout` parameter to specify a different value, including `-1` to set the timeout to infinity. For more information, see [Inactivity timeout](/docs/services/speech-to-text?topic=speech-to-text-input#timeouts-inactivity).
+-   A *session timeout* occurs if the service receives no data from the client or sends no interim results for 30 seconds. You cannot change the length of this timeout, but you can extend the session by sending the service any audio data, including just silence, before the timeout occurs. You must also set the `inactivity_timeout` to `-1`. You are charged for the duration of any data that you send to the service, including the silence that you send to extend a session. For more information, see [Session timeout](/docs/services/speech-to-text?topic=speech-to-text-input#timeouts-session).
```

Comment:
"And you are charged--for silence--" (never mind)

Reply:
Yes, isn't it great!

Reply:
In the naked logs I saw
Ten thousand sessions, maybe more...